/scratch/work/sync-sarge/src/client-api/src/c++/common/base/util/XMLProcessor.cpp: In
   static member function `static const BCHAR* 
   XMLProcessor::getElementAttributes(const BCHAR*, const BCHAR*, unsigned 
   int*, unsigned int*)':
/scratch/work/sync-sarge/src/client-api/src/c++/common/base/util/XMLProcessor.cpp:761: error: jump
   to label `finally'
/scratch/work/sync-sarge/src/client-api/src/c++/common/base/util/XMLProcessor.cpp:724: error: 
    from here
/scratch/work/sync-sarge/src/client-api/src/c++/common/base/util/XMLProcessor.cpp:727: error: 
    crosses initialization of `unsigned int l'

#
# Patch managed by http://www.holgerschurig.de/patcher.html
#

--- native/src/c++/common/base/util/XMLProcessor.cpp~compile-errors
+++ native/src/c++/common/base/util/XMLProcessor.cpp
@@ -719,12 +719,13 @@
     // example ot tag with attribute list
     // <body enc="base64">
     BCHAR *openTag = 0; //<tag
+    unsigned int l = 0;
     
     if (!xml) {
         goto finally;
     }
 
-    unsigned int l = bstrlen(tag);
+    l = bstrlen(tag);
 
     if(bstrcmp(tag, T("CDATA")) == 0) {
         goto finally;
